lib/rack/tracker/zanox/zanox.rb

```diff
@@ -7,11 +7,17 @@ class Mastertag < OpenStruct
 
   class Track < OpenStruct
     # Example: OrderID=[[DEFC-4321]]&CurrencySymbol=[[EUR]]&TotalPrice=[[23.40]]
+    # url_param gets passed into the url, but should not be one of main parameters set to zanox
     def write
-      to_h.map do |k,v|
+      events = to_h.delete_if { |k,v| k == :path_extension}
+      events.map do |k,v|
```

Comment:
something like `to_h.except(:path_extension)` should also produce the same result